karate-core/src/main/java/com/intuit/karate/StringUtils.java

```diff
@@ -222,4 +226,24 @@ public static String fixJavaScriptFunction(String text) {
         }
     }
 
+    public static class KStreamGobbler implements Runnable {
```

Comment:
utility method to invoke lighthouse on underlying OS.

Reply:
not sure I understand. is this a CLI ? do be aware of the utilities that already exist: https://stackoverflow.com/a/62911366/143475

Reply:
yeah - its pretty much same thing as Karate CLI support -